### ISSUE TITLE
Does this work for real-world 3D transformation between two different coordinate system?

### ISSUE THREAD

**Issue by jiafei1224** [nghiaho12/rigid_transform_3D#12]
If i have two coordinate system c1 and c2, and there are three sets of corresponding points that matchs in the real-world from both coordinate system, does this method work for finding transformation matrix? If yes, can you give me an example of such coordinates. Thanks

**Comment by nghiaho12** [comment]
Yes, but it's up to you to make sure all the points are in the same coordinate system. Else you will get a solution that doesn't make sense.
